test/common/router/router_ratelimit_test.cc

```diff
@@ -27,69 +28,54 @@ namespace Envoy {
 namespace Router {
 namespace {
 
-envoy::api::v2::route::RateLimit parseRateLimitFromJson(const std::string& json_string) {
+envoy::api::v2::route::RateLimit parseRateLimitFromV2Yaml(const std::string& yaml_string) {
   envoy::api::v2::route::RateLimit rate_limit;
-  auto json_object_ptr = Json::Factory::loadFromString(json_string);
-  Envoy::Config::RdsJson::translateRateLimit(*json_object_ptr, rate_limit);
+  MessageUtil::loadFromYaml(yaml_string, rate_limit);
```

Comment:
not an issue for this PR, but there is a combined function that does both load and validate. Maybe use that in the future?

Reply:
ah, `loadFromYamlAndValidate`. Noted.